### ISSUE TITLE
[Accessibility]: Repo Management - Can't Add Repositories using keyboard

### ISSUE THREAD

**Issue by jplahn** [sourcegraph/sourcegraph#36860]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33521

### Problem description

Using Firefox, I was unable to use tab to navigate to the "Add Repositories" button from the "Your repositories" page. I

### Expected behavior

Starting from the "Your repositories" page in Settings, I should be able to use tab to navigate to the "Add repositories" button 

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by sourcegraph-bot-2** [comment]
Heads up @jplahn - the "team/repo-management" label was applied to this issue.
